packages/mobile/src/navigator/Hamburger.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react'
+import { Line, Svg } from 'react-native-svg'
+
+const Hamburger = () => {
```

Comment:
I think @tarikbellamine also made an hamburger so eventually we can consolidate. 

also dont we usually prefer named function to arrow for standalone function?

Reply:
Indeed I did